services/graph/pkg/config/config.go

```diff
@@ -85,6 +87,11 @@ type Identity struct {
 	LDAP    LDAP   `yaml:"ldap"`
 }
 
+// API represents API configuration parameters.
+type API struct {
+	UserPatchLimit int `yaml:"user_patch_limit" env:"GRAPH_USER_PATCH_LIMIT" desc:"The amount of users allowed to be changed in PATCH requests."`
```

Comment:
IMO this would be better named `group_members_patch_limit`. With the description being something like: "the number of group members allowed to be added with a single patch request".

IIRC this really only works with adding members.

Reply:
That's fine, I'll change that, I wasn't really sure about the name to begin with, so I was hoping for some input. :)

Changed